android/engine/src/main/java/org/smartregister/fhircore/engine/data/local/register/RegisterRepository.kt

```diff
@@ -190,7 +193,7 @@ constructor(
       fhirPathDataExtractor
         .extractData(baseResource, fhirPathExpression)
         .takeWhile { it is Reference }
-        .pmap {
+        .map {
```

Comment:
Any reason for not using `pmap` here?

Reply:
We realized that it doesn't improve performance. We also need this to happen sequentially